documentation/source/user_section/installation/linux.rst

```diff
@@ -274,7 +274,7 @@ Procedure:
       PATH+=":$PWD/bins"
 
       # Download models & cie
-      mkdir data; pushd data; for x in PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models ; do sct_download_data -d $x; done; popd
+      mkdir data; pushd data; for x in PAM50 optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models deepreg_models ; do sct_download_data -d $x; done; popd
```

Comment:
This isn't just removing `gm_model`, it's also adding 3 others?

Ah, I see! This is making the documentation page line up with what's actually done in the `install_sct` script.

Reply:
Exactly!